conformance/tests/httproute-matching-across-routes.go

```diff
@@ -116,10 +116,32 @@ var HTTPRouteMatchingAcrossRoutes = suite.ConformanceTest{
 }
 
 func testName(tc http.ExpectedResponse, i int) string {
+
+	/* If TestCase name is provided then use that or else generate one */
+	if tc.TestCaseName != "" {
+		return tc.TestCaseName
+	}
+
 	headerStr := ""
-	if tc.Request.Headers != nil {
-		headerStr = " with headers"
+	reqStr := ""
+
+	/* Request to root path */
+	if tc.Request.Path == "/" {
+		if tc.Request.Headers != nil {
+			headerStr = "with headers"
+		}
+		reqStr = fmt.Sprintf("%d request %s", i, headerStr)
+	} else {
+		/* Request to non root path */
+		tc.Request.Path = "'" + tc.Request.Path + "'"
+		if tc.Request.Headers != nil {
+			headerStr = " with headers"
+		}
+		reqStr = fmt.Sprintf("%d request to %s%s%s", i, tc.Request.Host, tc.Request.Path, headerStr)
 	}
 
-	return fmt.Sprintf("%d request to %s%s%s should go to %s", i, tc.Request.Host, tc.Request.Path, headerStr, tc.Backend)
```

Comment:
Unfortunately this isn't quite working as I'd hoped:

```
TestConformance/HTTPRouteHostnameIntersection/HTTPRoutes_that_do_intersect_with_listener_hostnames/20_request_to_anotherwildcard.io'/s4'_should_go_to_404
```

I think it makes more sense just to put the path and hostname in quotes. It may be simpler if we replace all of this with:

``` 
fmt.Sprintf("%d request to '%s%s'%s should go to", i, tc.Request.Host, tc.Request.Path, headerStr)
```

We'd still need the fix for StatusCode support below, up to you how to integrate that.

Reply:
It would look like the below for root path.

 --- PASS: TestHTTPReqString/9_request_to_'/'_with_headers_should_go_to_404 (0.00s)
    --- PASS: TestHTTPReqString/5_request_to_'/'_with_headers_should_go_to_infra-backend-v1 (0.00s)

Non root path
    --- PASS: TestHTTPReqString/8_request_to_'/foo/try'_with_headers_should_go_to_infra-backend-v2 (0.00s)

Is that ok?